child/_child_data_from_HPC_to_AnVIL.Rmd

```diff
@@ -0,0 +1,60 @@
+If you're used to working on High Performance Computing (HPC, or "on-premises" institutional servers) you'll likely need to transfer files from your HPC to AnVIL. You can check out [this Terra documentation](https://support.terra.bio/hc/en-us/articles/10206456029083-How-to-install-gcloud-on-a-local-machine) for the most recent information on installing Google Cloud SDK. There's also great documentation about moving data to/from Google buckets [here](https://support.terra.bio/hc/en-us/articles/4409101169051-How-to-move-data-to-from-a-Google-bucket).
+
+You will need to:
+
+1. Install Google Cloud SDK (`gcloud`) on your server
+
+2. "Initialize" `glcoud` and authenticate with your Google account
+
+3. Connect to your Workspace
+
+4. Transfer files
+
+**Step 1:** Install Google Cloud SDK on your server. This is software that enables file transfer. The code here is an example specifically for a Linux server. You might need to follow different instructions [depending on your server setup](https://cloud.google.com/sdk/docs/install).
+
+```
+curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-linux-x86_64.tar.gz
+tar -xf google-cloud-cli-linux-x86_64.tar.gz
+google-cloud-sdk/install.sh
+```
+
+**Step 2:** Initialize `glcoud`. You might need to specify the path to the executable `gcloud`, e.g., `google-cloud-sdk/bin/gcloud init`.
+
```

Comment:
Is it `glcloud` or `gcloud`?

Reply:
(Line 21 says `glcloud`, line 24 says `gcloud`. Terra documentation says `gcloud`. I'm just going to make the change.)